css/main.css

```diff
@@ -1,3 +1,282 @@
-/* Add your CSS here */
+/* fonts
+font-family: 'Questrial', sans-serif;
+font-family: 'Raleway', sans-serif;
+font-family: 'Roboto', sans-serif;*/
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,400&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+li {
+    list-style: none;
+}
+
+a {
+    text-decoration: none;
+}
+.section {
+    display: flex;
+    margin-right: -2em;
+}
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1rem 1.5rem;
+}
+/* hamburger styles*/
+.hamburger {
+    display: none;
+}
+.bar {
+    display: block;
+    width: 30px;
+    height: 3.5px;
+    margin: 4.5px auto;
+    -webkit-transition: all 0.3s ease-in-out;
+    transition: all 0.3s ease-in-out;
+    background-color: #101010;
+}
+/*basic styles for our other elements*/
+.nav-menu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: -1em ;
+}
+
+.nav-item {
+    margin-left: 5rem;
+}
+
+.nav-link{
+    font-size: 1.6rem;
+    font-weight: 400;
+    color:  #475569;
+    color: #000;
+}
+
+.nav-link:hover{
+    color: #394B2A;
+}
+
+.bar:nth-child(2) {
+    opacity: 0;
+}
+.hamburger.active .bar:nth-child(2) {
+    opacity: 0;
+}
+
+.hamburger.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+}
+
+.hamburger.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+}
+.nav-menu {
+    position: fixed;
+    left: -100%;
+    top: 5rem;
+    flex-direction: column;
+    background-color: #fff;
+    width: 100%;
+    border-radius: 10px;
+    text-align: center;
+    transition: 0.3s;
+    box-shadow:
+        0 10px 27px rgba(0, 0, 0, 0.05);
+}
+
+.nav-menu.active {
+    left: 0;
+    top: 1em;
+}
+
+.nav-item {
+    margin: 1.1rem 0;
+}
+
+.hamburger {
+    display: block;
+    cursor: pointer;
+}
+
+
+
+/*my code*/
+body {
+    margin: 0;
+    font-family: 'Roboto', sans-serif;
+}
+header {
+    
+    font-family: 'Questrial', sans-serif;
+}
+a {
+    text-decoration: none;
+}
+.order {
```

Comment:
Look into managing button text with `white-space: nowrap`

Try breaking just this header out into a minimum example on Codepen and taking code _away_. It might make things clearer.

Reply:
Thanks Sally for the review. Super!
I will definitely try to correct and, most importantly, understand in practice all your comments written by you - and the experience of a more experienced reviewer - very important for me!